rollup.config.js

```diff
@@ -5,7 +5,7 @@ import analyzer from "rollup-plugin-analyzer";
 import replace from "@rollup/plugin-replace";
 import packageJson from "./package.json";
 
-const isProduction = process.env.NODE_ENV === "production";
+const isProduction = process.env.ROLLUP_ENV === "production";
```

Comment:
This appears to have been a bug. In package.json, it references `ROLLUP_ENV` rather than NODE_ENV